cmd/bosun/sched/template.go

```diff
@@ -41,6 +44,7 @@ func (s *Schedule) Data(rh *RunHistory, st *models.IncidentState, a *conf.Alert,
 		IsEmail:       isEmail,
 		schedule:      s,
 		runHistory:    rh,
+		ElasticHost:   es,
```

Comment:
Sure, I have added it for debug purpose. I will make the change as suggested.

I would also like your suggestion on logging for ES host map https://github.com/bosun-monitor/bosun/pull/1947/files#diff-2039e8e0c0f4873ec699c321c1e336f9R261, are okay with it?